src/components/FileNameInput.tsx

```diff
@@ -8,9 +8,10 @@ interface FileNameInputProps {
   onCancel: () => void;
   onSubmit: (newValue: string) => void;
   isNameValid: (name: string) => boolean;
+  className?: string;
 }
 
-export function FileNameInput({ fileName, isNameValid, onCancel, onSubmit }: FileNameInputProps) {
+export function FileNameInput({ className, fileName, isNameValid, onCancel, onSubmit }: FileNameInputProps) {
```

Comment:
I would rather not add a `className` prop to this component. You can update the padding here directly (and I have actually done it in #531 so we will have conflicts)

Reply:
Ok. Will revert my change and wait for your PR get approved.